spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -342,6 +356,20 @@ describe("kong start/stop #" .. strategy, function()
       end
     end)
 
+    if strategy == "cassandra" then
+      it("errors when cassandra contact points cannot be resolved", function()
+        local ok, stderr = helpers.kong_exec("start --prefix " .. helpers.test_conf.prefix, {
+          database = strategy,
+          cassandra_contact_points = "invalid.inexistent.host",
+          cassandra_keyspace = helpers.test_conf.cassandra_keyspace,
+        })
+
+        assert.False(ok)
+        assert.matches("could not resolve any of the provided Cassandra contact points " ..
+                       "(cassandra_contact_points = 'invalid.inexistent.host')", stderr, nil, true)
```

Comment:
A `finally` hook with: 

```lua
finally(function()
  helpers.kill_all()
  pcall(helpers.dir.rmtree)
end)
```

would be welcome and make this test more robust (what if the underlying behavior disappears, the test fails, and now we leave this test instance running? It could impact subsequent test cases).

Reply:
added finally.